IDE/Renesas/e2studio/GR-ROSE/common/user_settings.h

```diff
@@ -165,6 +174,7 @@
         #define WOLFSSL_RENESAS_TSIP_CRYPT
         #define WOLFSSL_RENESAS_TSIP_TLS
         #define WOLFSSL_RENESAS_TSIP_TLS_AES_CRYPT
+        #define NO_AES_192        /* TSIP can not handle 192bit key */
```

Comment:
Perhaps having `NO_AES_192` is always set in settings.h when `WOLFSSL_RENESAS_TSIP_TLS_AES_CRYPT` is defined?

Reply:
Since TSIP's wc_AesGcmEcnrypt/Decrypt replaced the S/W default function, I thought it should be banned except for key lengths that TSIP can handle. Is there any other good way to limit the key length?